SPECS/librsvg2/librsvg2.spec

```diff
@@ -1,64 +1,93 @@
+# https://github.com/rust-lang/rust/issues/47714
+%undefine _strict_symbol_defs_build
+
 # We want verbose builds
 %global _configure_disable_silent_rules 1
+
 # Use bundled deps as we don't ship the exact right versions for all the
 # required rust libraries
 %global bundled_rust_deps 1
+
```

Comment:
Extra request: since you're already here, could you remove these two macros and in the rest of the spec only leave the paths, which correspond to these macros being set to 1? I think we've got no intention to modify what these macros influence anyway and we'll have a shorter, cleaner spec. **Not** necessary for this PR, just an ask, if you have a minute.

Reply:
Done.

Reply:
Will do on next around clean up by separate PR.